app/features/margin.js

```diff
@@ -50,16 +50,14 @@ export function pushElement(els, direction) {
           ? payload.current - payload.amount
           : payload.current + payload.amount
       }))
-    .forEach(({el, style, margin}) =>
-      el.style[style] = `${margin < 0 ? 0 : margin}px`)
+    .forEach(({el, style, margin}) => el.style[style] = `${margin}px`)
 }
 
 export function pushAllElementSides(els, keycommand) {
   const combo = keycommand.split('+')
   let spoof = ''
 
   if (combo.includes('shift'))  spoof = 'shift+' + spoof
-  if (combo.includes('down'))   spoof = 'alt+' + spoof
```

Comment:
this breaks adjusting margin on all sides. what was the intended ux enhancement?

Reply:
adjusting margin on all sides happens with cmd + down or up arrow. It works fine. alt is used to negative margins and that one works fine too. Unless I'm missing something here.